app/javascript/components/ScheduledMessages.jsx

```diff
@@ -32,30 +212,45 @@ class ScheduledMessages extends Component {
         actions.updateSkill({
           ...skill,
           config: {
-            ...skill.config,
             schedule_type,
-            messages: []
+            messages: [
+              {id: uuidv4(), delay: 60, message: ''}
+            ]
           }
         })
       } else if (schedule_type == "fixed_time") {
         actions.updateSkill({
           ...skill,
           config: {
-            ...skill.config,
             schedule_type,
             messages: [
               {id: uuidv4(), schedule: '', message: ''}
             ]
           }
         })
+      } else if (schedule_type == "recurrent") {
+        this.setState({ dialogVisible: false })
+        actions.updateSkill({
+          ...skill,
+          config: {
+            schedule_type,
+            start_date: formatTimeISOWithTimezone(new Date()),
```

Comment:
When should snap to 0 seconds the result of `new Date()`. They won't be visible to the user in the UI.

I imagine that we would store as start_date the midnight time (in utc) of the user in his/her current locale. And then adding the `at` of each message.